framework/src/modules/chain/transport/transport.js

```diff
@@ -215,37 +215,32 @@ class Transport {
 	 */
 
 	/**
-	 * Description of blocks.
-	 *
-	 * @todo Add @param tags
-	 * @todo Add description of the function
+	 * Returns a set of full blocks starting from the ID defined in the payload up to
+	 * the current tip of the chain.
+	 * @param {object} payload
+	 * @param {string} payload.blockId - The ID of the starting block
+	 * @return {Promise<*|Promise<*>>}
 	 */
-	// eslint-disable-next-line consistent-return
-	async blocks(query) {
-		// Get 34 blocks with all data (joins) from provided block id
-		// According to maxium payload of 58150 bytes per block with every transaction being a vote
-		// Discounting maxium compression setting used in middleware
-		// Maximum transport payload = 2000000 bytes
-		if (!query || !query.lastBlockId) {
-			return {
-				success: false,
-				message: 'Invalid lastBlockId requested',
-			};
-		}
+	async getBlocksFromId(payload) {
+		const validationResult = validator.validate(
+			definitions.getBlocksFromIdRequest,
+			payload,
+		);
 
-		try {
-			const data = await this.blocksModule.loadBlocksFromLastBlockId(
-				query.lastBlockId,
-				34, // 1977100 bytes
+		if (validationResult.length) {
+			const err = validationResult;
+			const error = `${err[0].message}: ${err[0].path}`;
```

Comment:
Manu is trying to make strict usage of only `err` throughout the code.
Can we do:
```
const error = `${validationResult[0].message}: ${validationResult[0].path}`;
```

or 

```
const errList = validationResult;
const err = `${errList[0].message}: ${errList[0].path}`;
```

Reply:
then you can do below

```
this.logger.debug(
	{
		err,
		req: payload,
	},
        ...
);

throw new Error(err);
```